### PR TITLE
Fix is_running_in_databricks function

### PR DESCRIPTION
**Pull request by WeichenXu123**
Fix `is_running_in_databricks` function

### CODE REVIEW

File: johnsnowlabs/utils/env_utils.py [hunk 1]
```diff
@@ -104,12 +104,15 @@ def is_running_in_emr():
...
-    """Check if the currently running Python Process is running in Databricks or not
-    If any Environment Variable name contains 'DATABRICKS' this will return True, otherwise False
+    """ Check if the currently running Python Process is running in Databricks or not
     """
-    for k in os.environ.keys():
-        if "DATABRICKS" in k:
-            return True
+    if "IS_IN_DATABRICKS_MODEL_SERVING_ENV" in os.environ:
+        # Serving container installs apache/spark, not databricks runtime.
+        return False
```
Comment (harupy) [inline johnsnowlabs/utils/env_utils.py:111]: Do we need this?

File: johnsnowlabs/utils/env_utils.py [hunk 2]
```diff
@@ -104,12 +104,15 @@ def is_running_in_emr():
...
+
+    if "DATABRICKS_RUNTIME_VERSION" in os.environ:
+        return True
```
Comment (harupy) [inline johnsnowlabs/utils/env_utils.py:114]:
```suggestion
    return "DATABRICKS_RUNTIME_VERSION" in os.environ:
```

File: johnsnowlabs/utils/env_utils.py [hunk 3]
```diff
@@ -104,12 +104,15 @@ def is_running_in_emr():
 
 
 def is_running_in_databricks():
```
Comment (harupy) [inline johnsnowlabs/utils/env_utils.py:106]:
```suggestion
def is_running_in_databricks_runtime():
```

We also might want to rename this as well for clarity.

File: johnsnowlabs/utils/env_utils.py (hunk 1 above)
Comment (andrewshieh) [reply johnsnowlabs/utils/env_utils.py:111]: +1, would prefer not to expose the specific env variable names we have in Databricks model serving.

**Comment by C-K-Loan** [comment]
Thanks for the pr will be merged to next release @harupy  @WeichenXu123

**Pull request merged**